### PR TITLE
Improve testing/fstest

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Impact
fstest

## Testing
run fstest

### CODE REVIEW

File: testing/fstest/fstest_main.c [hunk 1]
```diff
@@ -969,7 +959,7 @@ static void show_useage(void)
   printf("Usage : fstest [OPTION [ARG]] ...\n");
   printf("-h    show this help statement\n");
   printf("-n    num of test loop e.g. [%d]\n", CONFIG_TESTING_FSTEST_NLOOPS);
-  printf("-m    mount point tobe test e.g. [%s]\n",
+  printf("-m    mount point to be test e.g. [%s]\n",
```
Comment (gustavonihei) [inline testing/fstest/fstest_main.c:962]:
```suggestion
  printf("-m    mount point to be tested e.g. [%s]\n",
```

Comment (xiaoxiang781216) [reply testing/fstest/fstest_main.c]: Done.

**Comment by xiaoxiang781216** [comment]
@gustavonihei do you have more suggestion?

**Review by gustavonihei — APPROVED** [review]

**Comment by gustavonihei** [comment]
> @gustavonihei do you have more suggestion?

No more comments, sorry for missing this PR. LGTM.

**Pull request merged**